File: gameOfLife/Cell.py
class Cell():
	def __init__(self):
		self.alive = False
	

	def isAlive(self):
		return self.alive
	
	def setAlive (self, booleanVal):
		self.alive = booleanVal
	
	def __str__(self):
		returnVal = "-"
		if self.alive :
			return "0"
		return returnVal

File: GameOfLifeJava/Cell.java

public class Cell {
	String defaultDisplay = "-";
	boolean setAlive = false;
	
	public Cell (){
		
		
	}
	
	public boolean isAlive (){
		return setAlive;
		
	}

	
	public void setAlive (boolean t){
		if (t == true){
			this.defaultDisplay = "0";
			
		}
		this.setAlive = t;
		
	}
	
	
	public String toString (){
		return defaultDisplay ;
		
	}
	
	
}
